### PR TITLE
Added Sysmon and Software Restriction Policies Event Logs

### PR DESCRIPTION
**Pull request by dstreefkerk**
Added Channels and Subscriptions for Sysinternals Sysmon and Software Restriction Policies Event Logs.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/windows-event-forwarding](https://github.com/palantir/windows-event-forwarding), @dstreefkerk! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

**Comment by cryps1s** [comment]
Thanks for reaching out @dstreefkerk. We'll take a look at this within the next few days. 

Appreciate the contribution! 

Best,
Dane

File: wef-subscriptions/Software-Restriction-Policies.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <MaxLatencyTime>300000</MaxLatencyTime>
+    </Batching>
+    <PushSettings>
+      <Heartbeat Interval="1800000"/>
+    </PushSettings>
+  </Delivery>
+  <Query><![CDATA[
+    <QueryList>
+      <Query Id="0" Path="System">
+        <Select Path="System">*[System[Provider[@Name='Microsoft-Windows-SoftwareRestrictionPolicies'] and (EventID=865 or EventID=866 or EventID=867 or EventID=868 or EventID=882)]]</Select>
```
Comment (Centurion89) [inline wef-subscriptions/Software-Restriction-Policies.xml]: If it's not too much trouble, can you add inline comments for each EventID similar to this?
 https://github.com/palantir/windows-event-forwarding/blob/ff2b753e85d3c48bcce313b445dd558be9a9984c/wef-subscriptions/Services.xml#L23-L24

**Comment by Centurion89** [comment]
Hey @dstreefkerk,

I have to apologize for this - we didn't foresee this issue when first open sourcing the repository, but would you be okay with removing CustomEventChannels.dll from this PR? It's a bit of a liability for us to host a .dll compiled by a third party. I test-compiled your updated manifest, but unfortunately the hash from my resulting dll didn't match yours (probably due to something small like different compiler versions or something).

Once your commit has been merged, we'll go ahead and upload a compiled .dll generated from your updated manifest and update our contribution guidance to account for this case.

**Comment by dstreefkerk** [comment]
Fair enough, definitely understand that.

I’ll sort it out as soon as I get a chance.

**Comment by dstreefkerk** [comment]
Hi guys, sorry for the delay on this. DLL is now removed.

**Comment by Centurion89** [comment]
No problem, thank you for contributing!

**Pull request merged**